python/lib/util/fs.py

```diff
@@ -77,3 +78,39 @@ def remove_empty_directories(dir_path: str):
     for subdir_path, _, _ in os.walk(dir_path, topdown=False):
         if is_directory_empty(subdir_path):
             os.rmdir(subdir_path)
+
+
+def get_file_extension(file_name: str) -> str:
+    """
+    Get the extension (including multiple extensions) of a file name or path without the leading
+    dot.
+    """
+
+    parts = file_name.split('.', maxsplit=1)
+    if len(parts) == 1:
+        return ''
+
+    return parts[1]
```

Comment:
```suggestion
    return parts[1:]
```

We definitely have .tar.gz in the pipeline for events